homeassistant/components/airnow/sensor.py

```diff
@@ -57,10 +58,10 @@ class AirNowEntityDescription(SensorEntityDescription, AirNowEntityDescriptionMi
 SENSOR_TYPES: tuple[AirNowEntityDescription, ...] = (
     AirNowEntityDescription(
         key=ATTR_API_AQI,
-        translation_key="aqi",
         icon="mdi:blur",
         native_unit_of_measurement="aqi",
         state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.AQI,
```

Comment:
It seems that an AQI sensor not can have a `native_unit_of_measurement`
https://developers.home-assistant.io/docs/core/entity/sensor/#available-device-classes